tasks/pruner_old_builds.yml

```diff
@@ -0,0 +1,48 @@
+- import_tasks: tasks/generic/serviceaccount.yml
```

Comment:
I wish there was a way to just reuse `tasks/pruner.yml` with different variable values. I don't know how to make that happen with an include though.

Reply:
Oh! you could use include_tasks. Maybe something like:

```
- include_tasks: pruner.yml
  with_items:
    - schedule: osbs_pruner_schedule_build
      cronjob_command: osbs_pruner_command_build
      name: build-pruner
    - schedule: osbs_pruner_old_builds_schedule_build
      cronjob_command: osbs_pruner_old_builds_command_build
      name: build-pruner-old-builds
  when: "osbs_pruner_serviceaccount is defined"

```

Then, just change this file to use `item.schedule`, `item.cronjob_command`, and `item.name`.